.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```

Comment:
- [OPTIONAL] I can see that you already added the `.vscode/` folder to .gitignore. However, the file is still present in your GitHub repository !! to Solve this small problem I suggest you to [Check this](https://medium.com/sela-devops-team/gitignore-doesnt-work-e1566845d30a). Or you can delete it manually from GitHub (another solution).